### DISCUSSION TITLE
Task "Hack the missile" Doesn't receive reinforcements

### DISCUSSION THREAD

**Discussion by Kiskadee-dev** [Vdauphin/HeartsAndMinds#1428] [General]
I'm not sure if this is a feature request or a bug report, so i'm opening this

The task of hacking the missile doesn't receive reinforcements after we reach the task to defend the position, so we just sit there and wait for the long time it takes without doing anything.

The enemies in the region doesn't notice and doesn't try to attack/defend the area, so there's no reason for this 'defend' task

**Comment by Vdauphin** [comment]
Hello,
You need to start the hacking, then enemies will come
Cheers
